exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/MetricAdapter.java

```diff
@@ -174,6 +178,54 @@ private static void addSummarySamples(
     }
   }
 
+  private static void addHistogramSamples(
+      DoubleHistogramPointData doubleHistogramPointData,
+      String name,
+      List<String> labelNames,
+      List<String> labelValues,
+      List<Sample> samples) {
+    samples.add(
+        new Sample(
+            name + SAMPLE_SUFFIX_COUNT,
+            labelNames,
+            labelValues,
+            doubleHistogramPointData.getCount()));
+    samples.add(
+        new Sample(
+            name + SAMPLE_SUFFIX_SUM, labelNames, labelValues, doubleHistogramPointData.getSum()));
+
+    List<String> labelNamesWithLe = new ArrayList<>(labelNames.size() + 1);
+    labelNamesWithLe.addAll(labelNames);
+    labelNamesWithLe.add(LABEL_NAME_LE);
+
+    long[] cumulativeCount = new long[] {0};
```

Comment:
Why do you need an array to only use the first value?

Reply:
variables used in lambda expression should be final or effectively final. so a primitive here won't do. the alternative would be `AtomicLong`. but not much difference.

Reply:
:)) can then avoid lambdas :)) I find this very strange. I saw the AtomicLong or the LongAdder

Reply:
updated. found a way to rewrite the loop without lambdas.

Reply:
ooo :) nice, there was life before lambdas :))